### PR TITLE
Moved files and made adjustments for navigation to be its own section

### PR DESCRIPTION
**Pull request by langsmith**
Fixes: https://github.com/mapbox/android-docs/issues/37

screen shot 2017-04-18 at 2 57 00 pm

### CODE REVIEW

File: pages/_template.jsx [hunk 1]
```diff
@@ -101,7 +102,9 @@ module.exports = React.createClass({
         {/* Site Navigation */}
         <div className={`align-t hmin48 flex-parent-inline flex-parent--center-cross flex-parent--left-main ${queryMatches.desktop ? 'w240' : ''}`}><a href={'https://mapbox.com/'}><div className={queryMatches.desktop ? 'ml12 mb-logo--s mb-logo--white' : ''}/></a></div>
         <Link className={`py12 transition btn color-white bg-transparent bg-darken10-on-active bg-darken10-on-hover txt-s  ${mapSdkActive ? 'is-active' : ''}`} to={prefixLink('/map-sdk/5.0.2/getting-started/')}>Map SDK</Link>
-        <Link className={`py12 transition btn color-white bg-transparent bg-darken10-on-active bg-darken10-on-hover txt-s  ${mapboxJavaActive ? 'is-active' : ''}`} to={prefixLink('/mapbox-services/2.1.0/getting-started/')}>Mapbox Services</Link>
+        <Link className={`py12 transition btn color-white bg-transparent bg-darken10-on-active bg-darken10-on-hover txt-s  ${mapboxJavaActive ? 'is-active' : ''}`} to={prefixLink('/mapbox-services/2.0.0/getting-started/')}>Mapbox Services</Link>
```
Comment (cammace) [inline pages/_template.jsx]: This should still link to 2.1.0, not 2.0.0

File: pages/navigation/2.1.0/navigation.md [hunk 2]
```diff
@@ -180,3 +180,9 @@ Not implemented in the SDK yet.
 <!-- preview -->
 
 The RouteUtils class can be found in the mapbox-java-services module and provides many of the methods used for calculations done in the RouteProgress object. An example of this is getting the total route distance left until the user reaches their destination. If you would like to do these calculations on your own or would like the change the behavior of navigation, you can directly call these methods and handle the calculations yourself.
+
+<br>
```
Comment (cammace) [inline pages/navigation/2.1.0/navigation.md]: any reason for these line breaks?

File: pages/_template.jsx (hunk 1 above)
Comment (langsmith) [inline pages/_template.jsx]: Fixed

File: pages/navigation/2.1.0/navigation.md (hunk 2 above)
Comment (langsmith) [inline pages/navigation/2.1.0/navigation.md]: @cammace, leaves some white space between the last line and the bottom of the browser window, which I thought looks a little better. No problem if you wanna' remove the breaks: 


screen shot 2017-04-18 at 5 17 57 pm

Without the breaks:

screen shot 2017-04-18 at 5 19 09 pm

Comment (cammace) [inline pages/navigation/2.1.0/navigation.md]: with the 2.1 PR, I added css padding to the layout for this purpose, you should be able to see it. In any case, better to do this in css then html.

Comment (langsmith) [inline pages/navigation/2.1.0/navigation.md]: Ok

**Pull request merged**